docs/Installation/Installation.rst

```diff
@@ -3,8 +3,141 @@
 Installation
 ************
 
-.. toctree::
-    :maxdepth: 1
+The best to install PYME will depend on your background and whether you are already using python on your computer.
+
+Executable installers (Windows-only)
+=====================================
+
+Recommended if you don't already have python on your computer and/or are unfamiliar with python. Download the latest installer from https://python-microscopy.org/downloads/. Double-click the installer and follow instructions. 
+
+**Caveats:** The executable installers lag the conda packages so you're getting an older version. The installers give you a bunch of error messages which you can safely ignore.
+
+
+Installing using conda
+======================
+
+Download and install the python **2.7** version of `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_.
+Then, open the *Anaconda prompt* [#anacondaprompt]_ and enter
+
+.. code-block:: bash
+	
+    conda config --add channels david_baddeley
```

Comment:
Do we not need the `anaconda` channel?

Reply:
The last time I nuked and paved the `anaconda` channel was there by default. Probably safer to add it.

Reply:
I also thought the last time I didn't need to add it.

Reply:
hmm, I though it was there for full anaconda, but not miniconda

Reply:
have added for now - we can always alter after some testing.